plugins/language_marko.lua

```diff
@@ -0,0 +1,63 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+-- Language Syntax References
+-- https://markojs.com/docs/syntax/
+
+syntax.add {
+    name = "Marko JS",
+    files = { "%.marko$" },
+    comment = "//",
+    block_comment = { "<!%-%-", "%-%->" },
+    patterns = {
+        { pattern = { "%-%-\n", "%-%-" },   type = "string" },   -- Multiline string
+        { pattern = "%-%-.-\n",             type = "string" },   -- Single line string
```

Comment:
```suggestion
        { pattern = "%-%-.*",               type = "string" },   -- Single line string
```
Where is this defined in the documentation website?

Reply:
For reference this is the documentation: https://markojs.com/docs/concise/